assets/styles.css

```diff
@@ -0,0 +1,399 @@
+}
+
+.caption {
+  color: #666;
+  font-size: 16px;
+}
+
+article header a {
+  font-size: 16px;
+}
+
+.article-description {
+  width: 50%;
+  margin: 30px auto;
+  position: relative;
+}
+
+.date-socials {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.fa-facebook,
+.fa-twitter,
+.fa-envelope,
+.fa-share {
+  background-color: #f5f5f5;
+  padding: 5px 10px;
+  border-radius: 15px;
+}
+
+.date {
+  font-size: 14px;
+}
+
+.section-title {
+  font-size: 20px;
+  margin-top: 20px;
+}
+
+.subscribe-section {
+  margin-top: 20px;
+  text-align: center;
+}
+
+.subscribe-section hr {
+  width: 90px;
+  margin: 20px auto;
+}
+
+.subscribe-section a {
+  font-size: 1em;
+  position: relative;
+  top: 3px;
+  color: #326891;
+}
+
+.aside-section {
+  position: absolute;
+  width: 220px;
+  top: 1700px;
+  right: 40px;
+}
+
+.inflation img {
+  margin-top: 10px;
+}
+
+.spiral-image img {
+  width: 500px;
+}
+
+.youtube-section a {
+  font-size: 20px;
+  color: #326891;
+}
+
+.footer-header img {
+  position: relative;
+  float: left;
+  width: 200px;
+}
+
+.footer-header a {
+  position: relative;
+  float: right;
+  text-align: right;
+  color: #567b95;
+  font-size: 12px;
+  width: 200px !important;
+}
+
+.editor-picks-article article img {
+  width: 80px;
+  margin-top: auto;
+}
+
+.editor-picks-article article {
+  display: flex;
+  margin: 10px auto;
+}
+
+.editor-picks-article article a {
+  padding: 10px;
+}
+
+.inflation {
+  width: 60vw;
+  margin: auto;
+  position: relative;
+}
+
+.inflation figure {
+  margin-top: 5px;
+}
+
+.thumbnail-texts {
+  position: absolute;
+  top: 14vw;
+}
+
+.thumbnail-texts section:nth-child(2) {
+  position: relative;
+  top: 6vw;
+}
+
+.thumbnail-texts p {
+  font-size: 0.7em;
+  width: 25%;
+  float: left;
+  padding: 8px;
+}
+
+.inflation hr {
+  width: 80px;
+}
+
+.inflation-footer {
+  font-size: 13px;
+}
+
+.youtube-section {
+  width: 40vw;
+  margin: 30px auto;
+}
+
+.youtube-section figure span {
+  color: #666;
+  font-size: 13px;
+}
+
+.spiral-image caption {
+  color: #666;
+  font-size: 10px;
+}
+
+.article-footer {
+  margin-top: 10px;
+}
+
+.article-footer small {
+  color: #666;
+}
+
+.more-section {
+  width: 90vw;
+  max-width: 1200px;
+  display: flex;
+  margin: 50px auto;
+}
+
+.grid-section h4,
+.popular h4 {
+  border-top: solid thin #333;
+  padding: 15px 0;
+}
+
+.grid-section {
+  flex-basis: 70%;
+}
+
+.grid-container {
+  display: -ms-grid;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 30px 15px;
+  margin-bottom: 30px;
+}
+
+.editors_picks {
+  margin-top: 30px;
+}
+
+.popular {
+  flex-basis: 30%;
+  padding-left: 20px;
+}
+
+.popular ul li {
+  list-style: none;
+  border-top: solid thin #e2e2e2;
+  padding: 15px 0;
+}
+
+.popular ul li a {
+  font-size: 19px;
+}
+
+.more-comments {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 10px;
+}
+
+.sub-footer {
+  display: flex;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_